scripts/banking/importFilteredOperations.js

```diff
@@ -11,14 +12,21 @@ const DOCTYPE_OPERATIONS = 'io.cozy.bank.operations'
  * @param {string} filepath The path to the JSON fixtures file
  * @param {string[]} categories A comma-separated list of category IDs
  * @param {number} nOperations Optional maximum number of records to import
+ * @param {boolean | string} deterministic Can be a seed, true to use the default order, or false for fully random
  */
```

Comment:
It would ease the reading to add a separator between the variable name and its description. Like `deterministic - Can be a seed, [...]`